cmd/query/app/querysvc/query_service.go

```diff
@@ -79,8 +80,8 @@ func (qs QueryService) GetServices(ctx context.Context) ([]string, error) {
 }
 
 // GetOperations is the queryService implementation of spanstore.Reader.GetOperations
-func (qs QueryService) GetOperations(ctx context.Context, service string) ([]string, error) {
-	return qs.spanReader.GetOperations(ctx, service)
+func (qs QueryService) GetOperations(ctx context.Context, service string, spanKind string) ([]*storage_v1.OperationMeta, error) {
```

Comment:
Please make it to accept parameters in a struct instead of positionally (context should remain separate).